draft-packages/tooltip/KaizenDraft/Tooltip/AppearanceAnim.tsx

```diff
@@ -23,7 +25,12 @@ export const AnimationContext = React.createContext<AnimationProps>({
  * When the component is no longer needed, it will no longer be rendered to the
  * dom.
  */
-export const AnimationProvider = ({ isVisible, ...otherProps }) => {
+export const AnimationProvider = ({
+  isVisible,
+  animationDuration = ANIM_DURATION_MS,
+  animationBuffer = ANIM_BUFFER,
+  ...otherProps
+}) => {
```

Comment:
I saw @dougmacknz had some comments about how this would need to be solved differently, but I'd also expect a test added for this change as this is a functional change to the component (it takes more props now).